common/reviews/api/rush-lib.api.md

```diff
@@ -582,55 +630,89 @@ export class RushConfigurationProject {
     get versionPolicyName(): string | undefined;
 }
 
+// @public
+export class RushConstants {
```

Comment:
There's a lot of random stuff in here. I think this should probably be `@beta` so we can clean it up.

Reply:
Good point, fixed